.github/workflows/dbt-build.yml

```
@@ -0,0 +1,27 @@
+name: Build and push dbt image
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - 'warehouse/**'
```

Always a good idea to include the workflow itself too:

```suggestion
      - '.github/workflows/dbt-build.yml'
      - 'warehouse/**'
```

---

oh yeah thanks, I will do that on the gtfs-rt-parser one too